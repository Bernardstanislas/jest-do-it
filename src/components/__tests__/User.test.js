import React from 'react';
import User from '../User';
import renderer from 'react-test-renderer';

test('User links to the user page', () => {
  const user = {
    login: 'jean_moust',
    avatarUrl: 'moustiswag.png',
    name: 'Jean De la Moustini√®re'
  };

  // Render a User component in the document
  const component = renderer.create(
    <User user={user} />
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});