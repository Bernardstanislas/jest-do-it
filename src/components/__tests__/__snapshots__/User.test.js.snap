// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User links to the user page 1`] = `
<div
  className="User"
>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    <img
      alt="jean_moust"
      height="72"
      src="moustiswag.png"
      width="72"
    />
    <h3>
      jean_moust
       
      <span>
        (
        Jean De la Moustini√®re
        )
      </span>
    </h3>
  </a>
</div>
`;
